commands/create_kubeconfig.go

```diff
@@ -364,7 +364,13 @@ func createKubeconfig(args createKubeconfigArguments) (createKubeconfigResult, e
 		createKubeconfigActivityName,
 		cmdLine)
 	if err != nil {
-		return result, microerror.Maskf(err, fmt.Sprintf("HTTP status: %d", apiResponse.StatusCode))
+		var errorMessage string
+		if apiResponse != nil {
+			errorMessage = fmt.Sprintf("HTTP status: %d", apiResponse.StatusCode)
+		} else {
+			errorMessage = err.Error()
```

Comment:
Can we make this more meaningful?

Reply:
Actually, this doesn't really make sense, as we mask the error later.